lang/tr.json

```
@@ -0,0 +1,746 @@
+{
+  "translationComplete": "1",
+  "version": {
+    "2kki": {
+      "label": "Yume 2kki Versiyon {VERSION}",
+      "substitutes": {
+        " Patch ": "Yama",
```

Shouldn't this be surrounded in spaces?
```suggestion
        " Patch ": " Yama ",
```